configurator/src/capture.rs

```diff
@@ -94,9 +103,21 @@ impl Capturer {
                 continue;
             }
             thread::sleep(Duration::from_millis(30));
-            let led_picture = self.camera.capture()?;
-
-            coords.push(cv::find_light_from_diff(&base_picture, &led_picture)?);
+            let mut led_picture = self.camera.capture()?;
+            let found_coords = cv::find_light_from_diff(&base_picture, &led_picture)?;
+            if save_pictures {
+                if found_coords.is_some() {
+                    led_picture.mark(found_coords.unwrap().0, found_coords.unwrap().1)?;
+                }
```

Comment:
```suggestion
                if let Some((x, y)) = found_coords {
                    led_picture.mark(x, y)?;
                }
```

Reply:
Thanks, I was pretty sure there was a better option for doing that 😁